src/logging/devPortalLogger.ts

```diff
@@ -0,0 +1,93 @@
+import { Bag } from "@paperbits/common";
+import { Logger } from "@paperbits/common/logging";
+import { HttpClient, HttpRequest, HttpHeader } from "@paperbits/common/http";
+import { ISettingsProvider } from "@paperbits/common/configuration";
+import { DevPortalEvent } from "../models/logging/devPortalEvent";
+import { v4 as uuidv4 } from 'uuid';
+import * as Constants from "./../constants";
+import { version } from "./../../package.json";
```

Comment:
I think we can't do that `import { version } from "./../../package.json";` because we make the dev configuration a part of application and that may have adverse effects on application bundling, tree-shaking. There is actually a special [webpack plugin](https://webpack.js.org/plugins/define-plugin/) for that purpose, so I'd recommend to use that.

Reply:
Thank you for the explanation, updated the way of retrieving the version from package json using this plugin. Applied only to runtime.